src/main/java/uk/gov/ons/ctp/response/collection/exercise/service/ActionRuleCreator.java

```diff
@@ -0,0 +1,13 @@
+package uk.gov.ons.ctp.response.collection.exercise.service;
+
+import uk.gov.ons.ctp.response.collection.exercise.domain.CaseTypeOverride;
+import uk.gov.ons.ctp.response.collection.exercise.domain.Event;
+import uk.gov.ons.response.survey.representation.SurveyDTO;
+
+public interface ActionRuleCreator {
+  void execute(
```

Comment:
I know public is default but as a general styling thing we seem to tend to be explicit about it (which I agree with)

Reply:
(this goes for a bunch of things through the PR, I'm not gonna bother tagging them all)